tutorials/nlp/Token_Classification-BioMegatron.ipynb

```diff
@@ -0,0 +1,602 @@
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we see the following tags within the abstract:\n",
+    "```html\n",
+    "<category=\"Modifier\">adenomatous polyposis coli tumour</category>\n",
+    "<category=\"Modifier\">adenomatous polyposis coli ( APC ) tumour</category>\n",
+    "<category=\"Modifier\">colon carcinoma</category>\n",
+    "<category=\"Modifier\">colon carcinoma</category>\n",
+    "<category=\"SpecificDisease\">cancer</category>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For our purposes, we will consider any identified category (such as \"Modifier\", \"Specific Disease\", and a few others) to generally be a \"disease\".\n",
+    "\n",
+    "Let's download the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_DIR = \"DATA_DIR\"\n",
+    "os.makedirs(DATA_DIR, exist_ok=True)\n",
+    "os.makedirs(os.path.join(DATA_DIR, 'NER'), exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Downloading NCBI data...')\n",
+    "wget.download('https://www.ncbi.nlm.nih.gov/CBBresearch/Dogan/DISEASE/NCBI_corpus.zip', DATA_DIR)\n",
+    "! unzip {DATA_DIR}/NCBI_corpus.zip -d {DATA_DIR}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to see more examples, you can explore the text of the corpus using the file browser to the left, or open files directly:\n",
+    "\n",
+    "* [DATA_DIR/NCBI_corpus_training.txt](DATA_DIR/NCBI_corpus_testing.txt)\n",
+    "* [DATA_DIR/NCBI_corpus_testing.txt](DATA_DIR/NCBI_corpus_testing.txt)\n",
+    "* [DATA_DIR/NCBI_corpus_development.txt](DATA_DIR/NCBI_corpus_development.txt)\n",
```

Comment:
You added these as links but they won't work - could you give an example of how to inspect the data with `head`?